### PR TITLE
Features/hdev 446 display tvl on vesting page

### PR DESCRIPTION
**Pull request by geonellov**
## Description of Changes
This PR displays vesting TVL on the TVL page

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-446)

### Developer Checklist:

* [X] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [X] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [ ] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [X] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

File: src/components/Tailwind/Cards/EventInfoCard.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                <Title>{content.title}</Title>
+                <CountDown>
+                  <div>
+                    <p>Days</p>
+                    <h2>{IfDoubleDigit(ConvertToString(countdown?.days)) ? `0${countdown?.days}` : countdown?.days}</h2>
+                  </div>
+                  <div>
+                    <p>Hours</p>
+                    <h2>
+                      <h2>
```
Comment (schystz) [inline src/components/Tailwind/Cards/EventInfoCard.tsx]: @geonellov unnecessary nested `<h2>` tags here

File: src/pages/HaloHalo/index.tsx [hunk 2]
```diff
@@ -302,22 +401,48 @@ export default function HaloHalo() {
...
+                anytime but lose out on monthly RNBW vesting multiples.
+              </RowBetween>
+            </CardSection>
+          </CardSectionContainer>
+        </ShowInSmall>
       </PageWrapper>
+      <EmptyPageWrapper>
+        <EmptyState header={t('emptyStateTitleInVest')} subHeader={t('emptyStateSubTitleInVest')} />
+      </EmptyPageWrapper>
+      {/*<div style={{ marginTop: '120px', height: '248px', maxWidth: '476px', width: '100%' }}>*/}
```
Comment (schystz) [inline src/pages/HaloHalo/index.tsx]: Can we remove the commented out codes here until line 445 if they are no longer needed?

File: src/components/Tailwind/Cards/EventInfoCard.tsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+import styled from 'styled-components'
+import { CurrentEpoch } from '../../../halo-hooks/useEpochCountdown'
+
+interface TimeObject {
+  event: boolean
+  content: Record<string, string>
+  countdown?: CurrentEpoch
+}
+
+const EventInfoCard = ({ event, content, countdown }: TimeObject) => {
```
Comment (schystz) [inline src/components/Tailwind/Cards/EventInfoCard.tsx]: Suggestion: rename to `EpochReleaseTimerCard` to be really specific

File: src/components/Card/GradientCard.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Card from './index'
+import styled from 'styled-components'
+
+const GradientCard = ({ children }: any) => {
+  const Wrapper = styled(Card)`
```
Comment (schystz) [inline src/components/Card/GradientCard.tsx]: For new components onwards, let's deprecate styled components and use tailwind css instead

File: src/components/Card/GradientCard.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    border-radius: 10px;
+    font-family: serif Open Sans;
+    font-style: normal;
+    color: #ffffff;
+    ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+       border-radius: 3px;
+    `};
+  `
+  return (
+    <>
```
Comment (schystz) [inline src/components/Card/GradientCard.tsx]: Can do without the additional `<></>` wrapping, we only need it when rendering texts or conditions

File: src/halo-hooks/useTVLInfo.ts [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import { ApolloClient, gql, InMemoryCache } from '@apollo/client'
+import { useCallback, useEffect, useState } from 'react'
+import { HALODAO_EXCHANGE_SUBGRAPH } from '../constants'
```
Comment (schystz) [inline src/halo-hooks/useTVLInfo.ts:3]: @geonellov looks like this env (`REACT_APP_HALODAO_EXCHANGE_SUBGRAPH_URL`) is not added to our github repo's secrets yet, the reason why TVL is blank on the preview page. Note that it needs to be on workflow yaml too in able to be recognized by the app.

File: src/halo-hooks/useTVLInfo.ts [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+        })
+      })
+      .catch(err => {
+        console.log('Error fetching data: ', err)
+      })
+  }, [])//eslint-disable-line
+
+  useEffect(() => {
+    fetchTVLInfo()
+  }, [fetchTVLInfo]) //eslint-disable-line
```
Comment (schystz) [inline src/halo-hooks/useTVLInfo.ts]: We might wanna refetch TVL too if network changes

```suggestion
  }, [fetchTVLInfo, chainId]) //eslint-disable-line
```

File: src/halo-hooks/useTVLInfo.ts (hunk 6 above)
Comment (geonellov) [reply src/halo-hooks/useTVLInfo.ts:3]: added in GH secret and deploy_dev.yml @schystz

**Comment by tracyarciaga** [comment]
@geonellov please fix the conflict for this PR

**Comment by geonellov** [comment]
> @geonellov please fix the conflict for this PR

@tracyarciaga fixed.

File: src/halo-hooks/useTVLInfo.ts (hunk 6 above)
Comment (schystz) [reply src/halo-hooks/useTVLInfo.ts:3]: It needs to be added on `deploy_preview.yml` as well @geonellov

Comment (geonellov) [reply src/halo-hooks/useTVLInfo.ts:3]: @schystz already added in PR https://github.com/HaloDAO/halodao-interface/pull/199

**Pull request merged**